### ISSUE TITLE
Accessibility: Navigation should not contain headings

### ISSUE THREAD

**Issue by sedlatschek** [laravel/nova-issues#3430]
- Laravel Version: 8.44.0
- Nova Version: 3.26.1
- PHP Version: 7.4.6
- Database Driver & Version: MySQL 5.7
- Operating System and Version: Win10
- Browser type and version: Chrome 88

### Description:
Headings are very important for accessibility. Screenreaders can be used to only read out headings, allowing the user the navigate freely between them. Headings are only expected to be used above content. Having headings in the navigation sidebar is confusing to blind users. Please replace the used headings (such as `h3`).

![image](https://user-images.githubusercontent.com/58665536/121051950-31e70400-c7ba-11eb-8524-7635b0929393.png)

Laravel Nova is not accessible. This is a problem in enterprise environments, where accessibility is a must requirement.

**Comment by davidhemphill** [comment]
👋 Hi there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.
